MANIFEST.in

```diff
@@ -14,8 +14,6 @@ graft requirements
 graft tests
 global-exclude *.pyc
 global-exclude *.cache
-exclude propcache/*.c
-exclude propcache/*.html
-exclude propcache/*.so
-exclude propcache/*.pyd
+exclude src/propcache/*.c
```

Comment:
I think this should be the opposite — avoid excluding intermediary C-files in sdists, but include pyx/pyd/py files so that tools could build wheels from sdists.

Reply:
Won't including the intermediate C-files in the source dist make the source dist much larger?

I'm confused why the binaries would be included in the source dist.

Reply:
Sorry, I should've been more verbose: exclude `*.so`, `*.c` and similar (`*.dll`, `*.dynlib` etc.); include `*.py`, `*.pyx`, `*.pxd`. It appears that my brain `*.pyd` and `*.pxd`. One is binary, and the other is source code.
You're right, though, exclude binaries, include sources. One day, we'll be able to hook setuptools-scm and this problem will disappear…